blog-cse/2024-04-11-application-update.md

```diff
@@ -0,0 +1,44 @@
+---
+title: April 11, 2024 - Application Update
+keywords:
+  - cloud siem
+  - mitre
+  - light mode
+  - dark mode
+tags: [application update]
+image: https://help.sumologic.com/img/sumo-square.png
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### MITRE ATT&amp;CK&reg; Explorer Enhancements
```

Comment:
```suggestion
### MITRE ATT&amp;CK&reg; Coverage Enhancements
```

Reply:
It's "MITRE ATT&CK Coverage" in the UI. We should align with that in the release notes.